src/controller/CHIPDeviceController.cpp

```diff
@@ -2925,6 +2994,31 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         }
     }
     break;
+    case CommissioningStage::kAttestationRevocationCheck: {
+        ChipLogProgress(Controller, "Verifying device's DAC chain revocation status");
+        if (!params.GetAttestationElements().HasValue() || !params.GetAttestationSignature().HasValue() ||
```

Comment:
This part, though, seems like pretty annoying code duplication.  Please have helper functions for "check that the params is ok for device attestation" and "create an AttestationInfo from params", if we're going to do it in multiple places.

Reply:
Addressed in latest commit